### PR TITLE
Feature Reopen cookie notice

### PR DESCRIPTION
**Pull request by andreasblumberg**
### Ticket
- [Reopen cookie notice](https://app.asana.com/0/809933051638353/1154380372326297/f)

### Description
-

### CODE REVIEW

File: dist/scripts/cookie-consent.js [hunk 1]
```diff
@@ -19,9 +25,10 @@
       return;
     }
 
-    var cookie_notice = document.getElementById('cookie-consent');
-    cookie_notice.removeAttribute('hidden');
-    document.body.classList.add('has-cookie-consent');
+    showCookieNotice();
+    var button = document.querySelector('[data-trigger-cookie-consent]');
+    console.log(button);
```
Comment (fabianmarz) [inline dist/scripts/cookie-consent.js]: debug code should be removed 😬 I also wonder why it's added in the dist script only 🤔 maybe added manually?

File: assets/scripts/cookie-consent.js [hunk 2]
```diff
@@ -3,21 +3,34 @@
     return;
   }
 
+  function addCookieSettingsButton() {
```
Comment (fabianmarz) [inline assets/scripts/cookie-consent.js:6]: The function name is misleading as this function doesn't actually add a button but registers the click handler for it. IMO this doesn't necessarily needs to be in a dedicated function as it's invoked once. But we can leave it to move forward 👍

File: dist/scripts/cookie-consent.js (hunk 1 above)
Comment (andreasblumberg) [reply dist/scripts/cookie-consent.js]: fixed

File: assets/scripts/cookie-consent.js (hunk 2 above)
Comment (andreasblumberg) [reply assets/scripts/cookie-consent.js:6]: referenced in issue and created a ticket for it to fast forward. #51

**Review by fabianmarz — APPROVED** [review]

**Pull request merged**